### DISCUSSION TITLE
Latest gluetun wll not start

### DISCUSSION THREAD

**Discussion by mikul9** [qdm12/gluetun#2374] [Q&A] [answered]
I updated to the latest image today and the container refuses to start. Oddly, I cannot even restart or remove the container except through Portainer. I've restarted Docker, restarted the server, and deleted the image and rebuilt the project, but it refuses to start. If there anything that I should do or is there an issue with the latest image?

**Comment by K4iN3R** [comment]
I have the same problem getting this error log:

"ERROR no iptables supported found: errors encountered are: iptables-nft: iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument (exit status 4); iptables: iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument (exit status 4)"

**Comment by mikul9** [comment]
I'm seeing the same error in the logs. I've now submitted an issue.

**Comment by jb2002** [comment]
I have the exact same issue

**Comment by ssolomon-tech** [comment]
Same here. Reverted to a previous release just to have a running container.

**Comment by qdm12** [comment, marked as answer]
This is fixed in 26705f5a23344b694baa55caaa420bf40017b54d
